eppo_metrics_sync/eppo_metrics_sync.py

```diff
@@ -51,14 +70,25 @@ def yaml_is_valid(yaml_path):
         for root, _, files in os.walk(self.directory):
             for file in files:
                 if file.endswith(".yaml") or file.endswith(".yml"):
+
                     yaml_path = os.path.join(root, file)
-                    valid = yaml_is_valid(yaml_path)
-                    if valid['passed']:
-                        self.load_yaml(yaml_path)
+
+                    if self.schema_type == 'eppo':
```

Comment:
I think this is fine if we only have two possible schema types, but I generally think dictionaries are cleaner for these cases.